### DISCUSSION TITLE
Display Unresponsive / Virtual Touchscreen Not Started

### DISCUSSION THREAD

**Discussion by focher** [tesla-android/issue-tracker#68] [General]
Hi,

I’ve definitely configured the fixed IP through Pihole so that the Tesla Android Pi is on 3.3.3.50. I can get the CarPlay / Autokit connection up fine and it’s visually in the browser, even reacting to things I do on the phone.

However, in the car’s browser the display is very laggy and doesn’t respond to any touch input whatsoever. I’m not sure how to troubleshoot.

Same thing was happening with 2022.25 and I’ve upgraded to 2022.27 with the same problem.

Any recommendations?

**Comment by mikegapinski** [comment]
What is you Tesla MCU version and firmware. I am running 2022.16.x on a Model 3. It works fine on Ryzen and Intel processors

**Comment by focher** [comment]
It’s a 2022 Model X built in June. We just got it. Firmware is 2022.16.3. 

I can try in my Model 3 too and report back. If there’s somewhere I can look at logs to troubleshoot, happy to do it.

**Reply by mikegapinski** [reply]
Hmm, I’ve seen reports on Twitter that it worked very well on Ryzen MX. How are you powering the pi inside the car? Maybe the cable doesn’t support power delivery over usb-c…

**Reply by mikegapinski** [reply]
Could you show me the output of 3.3.3.1:8001/state

and try pinging 3.3.3.50 via ssh on the Linux board?

**Comment by focher** [comment]
The good news is that I now have it working. I unplugged the network cable on the Android Pi, deleted the static IP address assignment, plugged it back in, then reassigned the 3.3.3.50 IP. Then it came up and worked.

Just as a reference, the /state endpoint had the following once it was working.

`{"ok": true, "result": { "encoder": {"type": "M2M-IMAGE", "quality": 80}, "source": {"resolution": {"width": 896, "height": 700}, "online": true, "desired_fps": 45, "captured_fps": 30}, "stream": {"queued_fps": 30, "clients": 1, "clients_stat": {"f615477169712968": {"fps": 30, "extra_headers": false, "advance_headers": false, "dual_final_frames": false, "zero_data": false, "key": "0"}}}}} `
